src/main/java/edu/ucsb/cs156/courses/services/UCSBCurriculumService.java

```diff
@@ -238,4 +241,31 @@ public String getJSONbyQtrEnrollCd(String quarter, String enrollCd) throws Excep
     log.info("json: {} contentType: {} statusCode: {}", retVal, contentType, statusCode);
     return retVal;
   }
+
+  public String getFinalsInfo(String quarter, String enrollCd) throws Exception {
+
+    HttpHeaders headers = new HttpHeaders();
+    headers.setAccept(Arrays.asList(MediaType.APPLICATION_JSON));
+    headers.set("ucsb-api-version", "3.0");
+    headers.set("ucsb-api-key", this.apiKey);
+
+    HttpEntity<String> entity = new HttpEntity<>("body", headers);
+
+    String params = String.format("?quarter=%s&enrollCode=%s", quarter, enrollCd);
+    String url = FINALS_ENDPOINT + params;
+
+    log.info("url=" + url);
+
+    String retVal = "";
+    MediaType contentType = null;
+    HttpStatus statusCode = null;
```

Comment:
```suggestion
    String retVal;
    MediaType contentType;
    HttpStatus statusCode;
```
Since these get immediately provided real values shortly afterward anyway, it doesn't make sense to give these fallback values like this. Leaving them declared but uninitialized is safer since it will provide a compile time warning if they are used before being provided their proper values.

Reply:
Otherwise LGM!

Reply:
Added your suggestion!